tests/unit/site.spec.ts

```diff
@@ -3,12 +3,28 @@ import { describe, it } from 'mocha';
 import { ObjectId } from 'mongodb';
 import { ChaiRequest, TEST_DB_CLIENT } from '../config';
 import { SiteModel } from '../../src/site/model';
+import { UserModel } from '../../src/user/model';
 
 require('../config'); // recommended way of loading root hooks
 
 const URL = '/sites';
 
 describe('# SITE', () => {
+  beforeEach(async () => {
```

Comment:
isnt this done in the config already?

Reply:
The Users table is not being done there. I felt it wasn't appropriate to clear out the Users table when hitting the reset endpoint, so I only wanted to clear out the users table on these tests.